src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-tue calendar--month-length-28">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--tue calendar--month-length--28">
```
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/f529ce8f-82ab-41c3-bc45-c30f99811fc1)